Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -88,6 +90,26 @@ extension PostgresQuery {
             self.sql.append(contentsOf: "$\(self.binds.count)")
         }
 
+
+        @usableFromInline
+        struct PostgresEncodableSequenceCannotBeEmpty: Error {
+            @usableFromInline init() { }
+        }
+
+        @inlinable
+        public mutating func appendInterpolation<C: Collection>(
+            _ values: C
+        ) throws where C.Element: PostgresThrowingDynamicTypeEncodable {
```

Comment:
We could also make this be more specific about the type just so there won't be accidental problems with someone not expecting their value to be inferred as a collection.
Maybe just have a function for Array and one for Set?

Reply:
Outdated now that i changed the function to have a parameter name.